README.md

```diff
@@ -1,2 +1,77 @@
-# comp150-agile
-Essay on the Agile Philosophy
+Should scrum be used to teach students for group projects instead of more traditional teaching methods in schools and higher education environments?
```

Comment:
Narrow the question. In it's current form, it will be too difficult to convincingly answer. Consider one particular aspect of scrum and only focus on secondary schools and the nature of projects they may conduct. A common criticism of school-level education, for example, is the lack of collaborative learning and teamwork; which, are skills that are frequently demanded by employers (see, e.g., [Higher Education and Work](http://eric.ed.gov/?id=ED394454)). Another might be [communication skills](http://search.proquest.com/openview/e7f152a2139d65aeefe6130e955863e8/1?pq-origsite=gscholar).